.github/workflows/pull_request_swift_matrix.yml

```diff
@@ -0,0 +1,84 @@
+name: Pull Request
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        type: string
+        description: "The name of the workflow used for the concurrency group."
+        required: true
+      matrix_linux_command:
+        type: string
+        description: "The command of the current Swift version linux matrix job to execute."
+        required: true
+      matrix_linux_5_8_enabled:
+        type: string
+        description: "The command of the 5.8 Swift version linux matrix job to execute."
```

Comment:
This and all the other `_enabled`s don't seem right, they should surely be booleans?

Reply:
actually, are at the `_enabled`s used at all?

Reply:
Thanks. Good catch!